posthog/api/query.py

```diff
@@ -78,6 +79,7 @@ def create(self, request, *args, **kwargs) -> Response:
             data.query = apply_dashboard_filters_to_dict(
                 data.query.model_dump(), data.filters_override.model_dump(), self.team
             )  # type: ignore
+        data.query.modifiers = create_default_modifiers_for_user(request.user)
```

Comment:
Double check this is ok. Motivation is to add the ability to query certain ways based on user specific feature flags